### DISCUSSION TITLE
TreeTable: when using sort options, is it possible to sort children as well?

### DISCUSSION THREAD

**Discussion by tsunamilx** [primefaces/community#1851] [PrimeVue] [answered]
Title

**Comment by tugcekucukoglu** [comment, marked as answer]
could you create an issue? It seems there is a bug.

**Comment by tsunamilx** [comment]
This was already reported https://github.com/primefaces/primevue/issues/1396 and was marked as completed, but it is not fixed, please take a look, thanks.
